test/semantics/stop.f90

```diff
@@ -0,0 +1,37 @@
+program main
```

Comment:
Needs more error examples for cases where the types are wrong, e.g. arrays and pointers.

Reply:
added